CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Revision history for llvm-pretty-bc-parser
 
+## 0.4.2.0 (August 2024)
+
+* Added support for GHC 9.8 and dropped official support of 9.2
+
+* Added support for new atomic operations in LLVM 9+.
```

Comment:
A minor thing: all of the previous changelog entries use present tense, so we may as well stick with that convention:

```suggestion
* Add support for GHC 9.8 and drop official support of 9.2.

* Add support for new atomic operations in LLVM 9+.
```